staging/src/k8s.io/api/core/v1/generated.proto

```diff
@@ -4512,6 +4530,12 @@ message SecurityContext {
   // This requires the ProcMountType feature flag to be enabled.
   // +optional
   optional string procMount = 9;
+
+  // The seccomp options to use by this container. If seccomp options are
+  // provided at both the pod & container level, the container options
+  // override the pod options.
+  // +optional
+  optional SeccompProfile seccompProfile = 11;
```

Comment:
Why is it field `11` and not `10`?

Reply:
This was automatically updated with `make update`. Going through the file there is already a number `10`:

https://github.com/kubernetes/kubernetes/blob/6115f15de19307871ff09ee950939f426b6ab56d/staging/src/k8s.io/api/core/v1/generated.proto#L4488